pinot-common/src/main/java/org/apache/pinot/common/messages/RunPeriodicTaskMessage.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.messages;
+
+import java.util.UUID;
+import javax.annotation.Nonnull;
+import org.apache.helix.ZNRecord;
+import org.apache.helix.model.Message;
+
+
+/**
+ * Upon receiving this message, Controller will execute the specified PeriodicTask against the tables for which it is
+ * the lead controller. The message is sent whenever API call for executing a PeriodicTask is invoked.
+ */
+public class RunPeriodicTaskMessage extends Message {
+  public static final String RUN_PERIODIC_TASK_MSG_SUB_TYPE = "PERIODIC_TASK";
+  private static final String PERIODIC_TASK_REQUEST_ID = "taskRequestId";
+  private static final String PERIODIC_TASK_NAME_KEY = "periodicTaskName";
+  private static final String TABLE_NAME_WITH_TYPE_KEY = "tableNameWithType";
```

Comment:
Seems all other messages put `tableName` as the key, should we try to match them?

Reply:
I had this q before, but our periodic tasks take table name with a type, and not raw table name. We can have two independent keys if you like

Reply:
All the user-defined messages always have table name with type (resource name) but use key `tableName`. I kind of prefer matching them, but either way is okay

Reply:
pending...

Reply:
@mcvsubbu @amrishlal Please take a look at `SegmentRefreshMessage` and `RoutingTableRebuildMessage` and see whether we want to match them